apisix/plugins/google-cloud-logging.lua

```diff
@@ -162,6 +163,7 @@ local function get_auth_config(config)
     end
 
     auth_config_cache = config_data
+    auth_config_cache.ssl_verify = conf.ssl_verify
```

Comment:
It seems the ssl_verify will be cached with the auth configuration. So changing the ssl_verify won't take effect immediately.

Reply:
this is a good question and has been updated.